releng/hu.elte.txtuml.target/mirrors.target

```diff
@@ -1,37 +1,29 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde version="3.8"?>
 <target name="mirrors" sequenceNumber="8">
-	<!-- TODO: change these to mirror sites for speeding up development -->
+	<!-- TODO: upload the same versions to the mirror site and change the locations in this file -->
```

Comment:
Yes, I am working on it... Basically I re-mirror 3 update sites to have the latest versions of these components. If I am right, xtext components could also be found in the mars repository, so we will not need a separate tmf update site.